### ISSUE TITLE
Fix contrast for breadcrumbs

### ISSUE THREAD

**Issue by nichgray** [CDRH/earlywashingtondc#246]
I adjusted the blue text generally, but I think it remains an issue for the breadcrumbs because of the gray background. See e.g. https://cdrhrailsdev.unl.edu/oscys/families/bell

**Comment by nichgray** [comment]
Should be resolved with https://github.com/CDRH/earlywashingtondc/pull/251/changes/fe79482e0d0ceda6e05502b3de332a135310ad83.
